types/index.d.ts

```diff
@@ -22,9 +22,7 @@ export interface Range {
   size: number
 }
 
-declare function defaultRangeExtractor(range: Range): number[]
-
-declare function useVirtual<T>(options: {
+export interface ReactVirtualOptions {
```

Comment:
Missing generic param `T`
```suggestion
export interface ReactVirtualOptions<T> {
```

Reply:
Also imho would drop the `ReactVirtual` prefix 

```suggestion
export interface Options<T> {
```

User can always rename it when importing, `import { useVirtual, Options as ReactVirtualOptions } from 'react-virtual'`

Reply:
yeah, don't know how I missed generic param :) vs code even highlight it!